lesson-6/twenty-one.js

```diff
@@ -34,22 +34,30 @@ function dealCard(deck) {
   } else if (randomSuit === 'spades') {
     card.unshift('♠️');
   }
-  return card;
+  hand.push(card);
 }
 
 function initialDeal(deck, playerHand, dealerHand) {
-  playerHand.push(dealCard(deck));
-  dealerHand.push(dealCard(deck));
-  playerHand.push(dealCard(deck));
-  dealerHand.push(dealCard(deck));
+  dealCard(deck, playerHand);
+  dealCard(deck, dealerHand);
+  dealCard(deck, playerHand);
+  dealCard(deck, dealerHand);
 }
 
 function hitOrStay() {
```

Comment:
extracted input verification from code body and included it within function itself